lib/membrane/rtp/outbound_packet_tracker.ex

```diff
@@ -65,8 +67,20 @@ defmodule Membrane.RTP.OutboundPacketTracker do
 
     {rtp_metadata, metadata} = Map.pop(buffer.metadata, :rtp, %{})
 
+    extensions =
```

Comment:
This whole statement seems like a lot of work that has to be done for each packet, please think how to optimize it a little bit, like `Map.keys`.

Reply:
When I think about it, in the end there will be not many element so I don't know if it is worth the effort in the end, preassigning `Map.keys` is probably enough here 🚀

Reply:
Guess we can resolve this ;)

Reply:
Yup, if you extract the `Map.keys` outside the filter :P

Reply:
Ah, sure 😝